config/module.config.php

```diff
@@ -18,18 +20,21 @@
         // 'class_name_scalars' => true,
     ],
     'service_manager'         => [
-        // Legacy Zend Framework aliases
         'aliases'   => [
-            \ZF\Configuration\ConfigResource::class        => ConfigResource::class,
-            \ZF\Configuration\ConfigResourceFactory::class => ConfigResourceFactory::class,
-            \ZF\Configuration\ConfigWriter::class          => ConfigWriter::class,
-            \ZF\Configuration\ModuleUtils::class           => ModuleUtils::class,
+            // Legacy Zend Framework aliases
+            \ZF\Configuration\ConfigResource::class  => ConfigResource::class,
+            \ZF\Configuration\ResourceFactory::class => ResourceFactory::class,
```

Comment:
But this removes the reference for `\ZF\Configuration\ConfigResourceFactory::class`.

Reply:
> Obviously it doesn't create `ConfigResourceFactory`, but `ResourceFactory `instead.

But if someone uses the wrong but original name to fetch the service, this will no longer work.

Reply:
Ok, I see your point. Someone might have done `$container->get(ConfigResourceFactory::class)`, got `ResourceFactory` instead and is happy with that.
I'll get back the reference and add a TODO to delete it in the next major release.